stdeb/util.py

```diff
@@ -975,13 +975,16 @@ def __init__(self,
 
         self.dirlist = ""
 
-        if not (with_python2 or with_python3):
-            raise RuntimeError('nothing to do - neither Python 2 or 3.')
-        sequencer_with = []
-        if with_python2:
-            sequencer_with.append('python2')
-        if with_python3:
-            sequencer_with.append('python3')
+        # check
+        if not (with_python2 ^ with_python3):
```

Comment:
Don't use binary operators here, `or` should do the trick. However, I'm not really following why we need the python2/python3 options, especially if we are not passing the flags to the dh-virtualenv to pick the correct Python version.